doc/Navigation/HowTo-DisplayMessageDialog.md

```diff
@@ -0,0 +1,55 @@
+# How-To: Display a Message Dialog
+
+- Add button to MainPage 
+
+```xml
+<Button Content="Show Simple Message Dialog"
+                    Click="{x:Bind ViewModel.ShowSimpleDialog}" />
+```
+
+- Add method to MainViewModel
+
+```csharp
+	public async Task ShowSimpleDialog()
```

Comment:
For final tidying up, aligning snippets to the left will look nicer.

Reply:
Agreed and thanks for the review - this is currently just a dump of content that needs reworking anyhow